### DISCUSSION TITLE
Changes made on github are not synced with codesandbox

### DISCUSSION THREAD

**Discussion by poioit** [codesandbox/codesandbox-client#5341] [Q&A]
I just imported my github project to codesandbox on the web.
I have some commits outside CodeSandbox directly to github on the same branch (master), the sandbox stop syncing.
Checked issues this should be fixed before. Don't know if this failed again? 

Link to sandbox:
https://codesandbox.io/s/github/poioit/cra-app

**Comment by garethx** [comment]
Which commits aren't being included? From an initial look it seems like they're all there.

**Reply by poioit** [reply]
This is the project I am using.
https://github.com/poioit/cra-app/commits/master
I've push 4 commits to test this.
But the codesandbox keeps on the first commit.

**Comment by garethx** [comment]
Are you looking at the project at https://codesandbox.io/s/github/poioit/cra-app? Only I see 'Hello, lets's start!!' for example, which is the latest commit in https://github.com/poioit/cra-app/commits/master

**Reply by poioit** [reply]
sorry, I thought I've commit some changes to import react. Apparently, I didn't commit that. It's a silly mistake I made. It definitely works. Thanks for your reply.
